doc/ale.txt

```diff
@@ -946,6 +947,36 @@ ALEWarningSign                                                 *ALEWarningSign*
   The highlight used for warning signs. See |g:ale_set_signs|.
 
 
+-------------------------------------------------------------------------------
+3.2. Options for write-good Linter                     *ale-write-good-options*
```

Comment:
You should either write "options for write-good," or "options for the write-good linter." The latter sounds weird without the definite article. The former sounds better.